app/src/main/java/org/jellyfin/androidtv/ui/preference/custom/RichListDialogFragment.kt

```diff
@@ -29,7 +34,8 @@ class RichListDialogFragment : LeanbackPreferenceDialogFragmentCompat() {
 
 	private fun <K> RichListPreference<K>.createAdapter() = Adapter(
 		items = getItems(),
-		selectedValue = value
+		selectedValue = value,
+		isSubtitle = false
```

Comment:
I'm not a fan of the changes to this file. I think it might be better to create a new preference UI type for color selection.

Reply:
Sure thing, perhaps ColorListDialogFragment?

Reply:
Maybe ColorPickerDialogFragment? If we ever change it from a list of colors to, for example, RGB sliders the name would still work.